external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/EmptyKafkaTupleListener.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.storm.kafka.spout;
```

Comment:
You need to add the Apache license to the top of the new files. You can just copy it from one of the other ones.